### PR TITLE
Buildscript update

### PR DESCRIPTION
**Pull request by alcatrazEscapee**
- Update for new plugin syntax in FG5
- Fix JUnit not loading in gameTestServer, removes a bunch of redundant crap code.
- Tweak code style - just change the import rules for simplicity, and with the goal of allowing `import static junit.*` imports without these getting replaced with individual imports by auto reformat all the time.

A couple user facing bug fixes:

- Fixes vessel infinite expiry prolonging bug
- Add more recipes for boiling eggs
- Add missing aqueduct item tag.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -84,29 +62,32 @@ repositories {
...
-            property 'forge.logging.console.level', 'debug'
-            property 'mixin.env.remapRefMap', 'true'
-            property 'mixin.env.refMapRemappingFile', "${projectDir}/build/createSrgToMcp/output.srg"
-            property 'forge.enabledGameTestNamespaces', mod_id
-            jvmArgs('-ea', '-Xmx4G', '-Xms4G')
+        all {
+            args("-mixin.config=${mod_id}.mixins.json")
+
+            property("forge.logging.console.level", "debug")
+            property("forge.enabledGameTestNamespaces", mod_id)
```
Comment (eerussianguy) [inline build.gradle:77]: should this be in all?

File: build.gradle [hunk 2]
```diff
@@ -119,83 +100,57 @@ minecraft {
             }
         }
 
+        client {
+            workingDirectory project.file("run/client")
```
Comment (eerussianguy) [inline build.gradle]: why not just run?

Comment (alcatrazEscapee) [reply build.gradle]: Thought was so it's clear we have run/client, run/server, and run/gametest. i.e. any individual files belong to either client, server, or gametest (no files are just in run, not a priori knowing what process they came from)

File: build.gradle (hunk 1 above)
Comment (alcatrazEscapee) [reply build.gradle:77]: Yes, to enable /test in run client / server. The difference between test server and gametest server is just that the latter is for CI.

**Pull request merged**